linode/manager
# test: [M3-8546] - Add unit tests for `DialogTitle` component

## Description 📝
Added Unit test cases for `DialogTitle` components.

## Changes  🔄

- Add unit test cases for `DialogTitle` components.

## Target release date 🗓️
N/A

## How to test 🧪

### Verification steps
(How to verify changes)
- Run `yarn test src/components/DialogTitle --reporter verbose`
- Verify that all the test cases pass.

## As an Author I have considered 🤔

*Check all that apply*
